### PR TITLE
chore: fix typos [skip ci]

### PR DESCRIPTION
**Pull request by Thabokani**
## Change Description
fix some typos

### Code Style and Documentation
- [x] The change obeys the [Code Documentation and Commenting](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#CodeDocumentation) guidelines, and lines wrap at 80.
- [x] Commits follow the [Ideal Git Commit Structure](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md#IdealGitCommitStructure).
- [ ] Any new logging statements use an appropriate subsystem and logging level.
- [ ] Any new lncli commands have appropriate tags in the comments for the rpc in the proto file.
- [ ]  [There is a change description in the release notes](https://github.com/lightningnetwork/lnd/tree/master/docs/release-notes), or `[skip ci]` in the commit message for small changes.

📝 Please see our  [Contribution Guidelines](https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md) for further guidance.

### CODE REVIEW

**Review by yyforyongyu — CHANGES_REQUESTED** [review]
Thanks for the PR! Could you squash your commits into a single one?

**Comment by Thabokani** [comment]
@yyforyongyu I have squashed two directories into one commit. Is that right? thanks for your review.

**Comment by Thabokani** [comment]
updated for `lint code`

File: kvdb/config.go [hunk 1]
```diff
@@ -35,7 +35,7 @@ const (
 
 // BoltConfig holds bolt configuration.
 type BoltConfig struct {
-	NoFreelistSync bool `long:"nofreelistsync" description:"Whether the databases used within lnd should sync their freelist to disk. This is set to true by default, meaning we don't sync the free-list resulting in imporved memory performance during operation, but with an increase in startup time."`
+	NoFreelistSync bool `long:"nofreelistsync" description:"Whether the databases used within lnd should sync their freelist to disk. This is set to true by default, meaning we don't sync the free-list resulting in improved memory performance during operation, but with an increase in startup time."`
```
Comment (guggero) [inline kvdb/config.go:38]: This should cause a change in the top level `sample-lnd.conf` file as well I think?

**Review by guggero — APPROVED** [review]
Thanks, LGTM :tada:

**Review by yyforyongyu — APPROVED** [review]

File: kvdb/config.go (hunk 1 above)
Comment (Thabokani) [reply kvdb/config.go:38]: I found that `config.go` and `sample-lnd.conf` were not exactly the same, and there is no typo error in `sample-lnd.conf`

**Pull request merged**